app/src/main/java/camp/nextstep/edu/calculator/CalculatorPresenterInterface.kt

```diff
@@ -0,0 +1,11 @@
+package camp.nextstep.edu.calculator
+
+import com.nextstep.edu.domain.Operator
+
+interface CalculatorPresenterInterface {
```

Comment:
PresenterInterface와 CalculatorViewInterface가 따로 설정되어져 있는데요!
MVP 아키텍쳐의 기본 형식에 맞춰 Contract로 묶어주는 것을 권장드립니다.
```suggestion
interface CalculatorContract {
    interface View {
        ...
    }
    
    interface Presenter {
        ...
    }
}
```

추가로 한가지 더 이야기하자면..! 변수명, 함수명, 클래스명, 인터페이스명 등등 네이밍에 타입, 형식등이 포함되지 않는 것이 좋습니다.
이 것은 추후 변경 가능성과 관련이 깊은데요~
물론 CalculatorPresenterInterface는 MVP 아키텍쳐용으로 설계되어있는 presenter이기 때문에 다른 형식으로 변할일이 없지만, 일부의 구현이 포함된 클래스로 변경해야하는 상황이 발생할 수 있는데요~ 이 경우 abstract class로 변경해야하고, 이 경우 네이밍을 같이 변경해야하는 이슈가 발생합니다.
또 다른 예시로, `val resultList: List<Result>`의 경우도 리스트가 아닌 Array의 형식으로 바꿔야한다면 `resultArray`로 바꿔야하기때문에 `results`같이 복수형을 활용해주는 것이 좋습니다.

Reply:
저도 인터페이스명에 Interface를 안쓰는 것으로 알고 있어서 다른 이름이 뭐가 있을까 고민 고민하다 CalculatorPresenter은 이미 있어서 어쩔 수 없이 썼는데.. Contract로 묶는군요!!! 감사합니다!